apps/server/src/modules/files/uc/delete-files.uc.ts

```diff
@@ -1,21 +1,22 @@
 /* eslint-disable no-await-in-loop */
 import { DeleteObjectCommand, S3Client } from '@aws-sdk/client-s3';
 import { Injectable } from '@nestjs/common';
-import { File, StorageProvider } from '@shared/domain';
-import { FilesRepo } from '@shared/repo';
-import { StorageProviderRepo } from '@shared/repo/storageprovider/storageprovider.repo';
-import { LegacyLogger } from '@src/core/logger/legacy-logger.service';
+import { StorageProviderEntity } from '@shared/domain';
+import { StorageProviderRepo } from '@shared/repo/storageprovider';
+import { LegacyLogger } from '@src/core/logger';
+import { FilesRepo } from '../repo';
+import { FileEntity } from '../entity';
 
 @Injectable()
-export class DeleteFilesUc {
+export class DeleteFilesUC {
```

Comment:
its a bit inconsistent, but the vast majority of our usecases are called "Uc" and not "UC"

Reply:
Ok sure, I'll change that.